charts/rancher-monitoring/v0.0.8/charts/operator/templates/deployment.yaml

```diff
@@ -48,7 +48,6 @@ spec:
             - --prometheus-config-reloader={{ template "system_default_registry" . }}{{ .Values.image.prometheusConfigReloader.repository }}:{{ .Values.image.prometheusConfigReloader.tag }}
             - --config-reloader-image={{ template "system_default_registry" . }}{{ .Values.image.configmapReload.repository }}:{{ .Values.image.configmapReload.tag }}
             - --labels={{ .Values.apiGroup }}=true
-            - --crd-apigroup={{ template "operator_api_group" . }}
```

Comment:
Why was this dropped?

Reply:
they dropped the flag in their release. @deniseschannon actually don't merge it, there are some possible chart changes regarding to CRD.